test/kubemark/stop-kubemark.sh

```diff
@@ -17,6 +17,8 @@
 
 # Script that destroys Kubemark cluster and deletes all master resources.
 
+export ENABLE_APISERVER_INSECURE_PORT="${ENABLE_APISERVER_INSECURE_PORT:-false}"
```

Comment:
KUBE_ENABLE_APISERVER_INSECURE_PORT

Reply:
will address along with the decision 1) name of the variable to controller the insecure/secure port  2) two variables: one for enable insecure port, one for use insecure ports in kubemarktest